scripts/feature_tests/boot_iso/boot_iso_vars.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export NUM_OF_CONTROLPLANE_REPLICAS="1"
+export NUM_OF_WORKER_REPLICAS="1"
```

Comment:
These are set in boot iso specific Makefile, no need to redefine them here

Reply:
Yes this is not necessary and can be removed